common/c_cpp/src/c/linux/port.h

```diff
@@ -197,7 +197,7 @@ const char* getHostName (void);
 #      define MAMADeprecated__(MSG)     __attribute__ ((deprecated(#MSG)))
 #    endif
 /* GCC 4.5 and greater also supports message with deprecated */
-#  elif (__GNUC__ >= 4 && __GNUC_MINOR__ >= 5)
+#  elif (__GNUC__ >= 4 && __GNUC_MINOR__ >= 5) || __GNUC__ > 4
```

Comment:
Does this not start to include GCC versions 4 - 4.4 inclusive now? 
If i recall correctly (and judging my the comment above) the __attribute__ calls may only be compatible on gcc 4.5 onwards?